src/lib/components/Identifiers/IdentifiersField.js

```diff
@@ -35,26 +34,24 @@ export class IdentifiersField extends Component {
         >
           {({ array, arrayHelpers, indexPath, key }) => (
             <GroupField widths="equal">
-              {schemeOptions &&
+              {schemeOptions && (
```

Comment:
question: mostly for my learning, does the parenthesis imply any change on behaviour?

Reply:
No, it is again Prettier with ESLint with the `create-react-app` rules, which should be the most widely used in the JS community.